# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/disable-google-api-warning.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-136.0.7103.33/chrome/browser/ui/startup/infobar_utils.cc.vanilla	2025-04-20 16:02:44.020170479 +0200
+++ chromium-136.0.7103.33/chrome/browser/ui/startup/infobar_utils.cc	2025-04-20 16:03:22.089640429 +0200
@@ -156,9 +156,9 @@
     infobars::ContentInfoBarManager* infobar_manager =
         infobars::ContentInfoBarManager::FromWebContents(web_contents);
 
-    if (!google_apis::HasAPIKeyConfigured()) {
-      GoogleApiKeysInfoBarDelegate::Create(infobar_manager);
-    }
+    //if (!google_apis::HasAPIKeyConfigured()) {
+    //  GoogleApiKeysInfoBarDelegate::Create(infobar_manager);
+    //}
 
     if (ObsoleteSystem::IsObsoleteNowOrSoon()) {
       PrefService* local_state = g_browser_process->local_state();
